README.md

```diff
@@ -83,12 +83,12 @@ br.com.roxana.example.InsufficientFundsException = Saldo Insufficiente: {funds}.
 
 @ResponseStatus(code = OK)
 @RequestMapping(method = POST, path = "/{id}/cars/")
-// Roxana provide a wrapper object that all reste responses should use.
+// Roxana provide a wrapper object that all rest responses should use.
 // This make the api more standardized.
 public @ResponseBody Response<Message> buyACar(@PathVariable("id") Long idPerson, 
               @RequestBody Car car) throws InsufficientFundsException {
 
-    // The code below is not in a service layer because it for demo proporse.
+    // The code below is not in a service layer because it is for a demo purpose.
```

Comment:
"is for demo purpose" acredito que nao precisa "a".

Reply:
:+1: